static/js/components/search/CountryRefinementOption.js

```diff
@@ -10,17 +10,22 @@ export default class CountryRefinementOption extends React.Component {
     count:    number,
   };
 
-  countryName = (countryCode: string): string => (
-    iso3166.country(countryCode).name
-  );
+  countryName = (countryCode: string): string => {
+    let country, countryName = "-";
+    if (countryCode) {
+      country = iso3166.country(countryCode);
+      countryName = country ? country.name : "-";
+    }
+    return countryName;
```

Comment:
the issue is this iso3166.country(countryCode), it will crash if ``countryCode`` is not valid.

```js
TypeError: Cannot read property 'trim' of null
      at Object.functions.country (node_modules/iso-3166-2/iso3166.min.js:9:17511)
      at CountryRefinementOption.__countryName__REACT_HOT_LOADER__ (static/js/components/search/CountryRefinementOption.js:15:44)
      at CountryRefinementOption._this.countryName (static/js/components/search/CountryRefinementOption.js:46:65)
      at CountryRefinementOption.render (static/js/components/search/CountryRefinementOption.js:34:18)
      at node_modules/react-dom/lib/ReactCompositeComponent.js:796:21
      at measureLifeCyclePerf (node_modules/react-dom/lib/ReactCompositeComponent.js:75:12)
      at ReactCompositeComponent._renderValidatedComponentWithoutOwnerOrContext (node_modules/react-dom/lib/ReactCompositeComponent.js:795:25)
      at ReactCompositeComponent._renderValidatedComponent (node_modules/react-dom/lib/ReactCompositeComponent.js:822:32)
      at ReactCompositeComponent.performInitialMount (node_modules/react-dom/lib/ReactCompositeComponent.js:362:30)
      at ReactCompositeComponent.mountComponent (node_modules/react-dom/lib/ReactCompositeComponent.js:258:21)
      at Object.ReactReconciler.mountComponent (node_modules/react-dom/lib/ReactReconciler.js:46:35)
      at ReactCompositeComponent.performInitialMount (node_modules/react-dom/lib/ReactCompositeComponent.js:371:34)
      at ReactCompositeComponent.mountComponent (node_modules/react-dom/lib/ReactCompositeComponent.js:258:21)
      at Object.ReactReconciler.mountComponent (node_modules/react-dom/lib/ReactReconciler.js:46:35)
      at mountComponentIntoNode (node_modules/react-dom/lib/ReactMount.js:104:32)
      at ReactReconcileTransaction.TransactionImpl.perform (node_modules/react-dom/lib/Transaction.js:140:20)
      at batchedMountComponentIntoNode (node_modules/react-dom/lib/ReactMount.js:126:15)
      at ReactDefaultBatchingStrategyTransaction.TransactionImpl.perform (node_modules/react-dom/lib/Transaction.js:140:20)
      at Object.ReactDefaultBatchingStrategy.batchedUpdates (node_modules/react-dom/lib/ReactDefaultBatchingStrategy.js:62:26)
      at Object.batchedUpdates (node_modules/react-dom/lib/ReactUpdates.js:97:27)
      at Object.ReactMount._renderNewRootComponent (node_modules/react-dom/lib/ReactMount.js:320:18)
      at Object.ReactMount._renderSubtreeIntoContainer (node_modules/react-dom/lib/ReactMount.js:401:32)
      at Object.ReactMount.render (node_modules/react-dom/lib/ReactMount.js:422:23)
      at makeStrippedHtml (static/js/util/util.js:243:14)
      at renderCountryOption (static/js/components/search/CountryRefinementOption_test.js:14:5)
      at Context.<anonymous> (static/js/components/search/CountryRefinementOption_test.js:37:18)


```